src/components/ListingCard/index.tsx

```diff
@@ -83,6 +91,12 @@ export default function ListingCard({
       $selected={isSelected}
       onClick={() => onClick?.(listing.id)}
     >
+      {interpretation && (
+        <Styles.LanguageSupportLabel>
+          {interpretationType}
+        </Styles.LanguageSupportLabel>
+      )}
```

Comment:
nit:  i interpreted `interpretation` as a differentiator between Case and Case Int, rather than a differentiator between Language Support and non-language support. In Listing Details, I render the interpretation type label based on DOC/INT/(CASE && interpretation). Do we want to modify this (or i can modify listingDetails) to make it consistent?

Reply:
so true, will change

Reply:
i think my interpretation was that `interpretation` indicated whether it should be rendered as a "language support type" listing card (i.e. has a blue tag)

we can probably leave this as is since we're fixing the prop for the page